modules/ROOT/pages/modularizing-your-configuration-files-for-team-development.adoc

```diff
@@ -4,51 +4,44 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint studio, team, shared projects
 
-Though it may seem convenient to keep all your Mule configuration in one place,
-the reality is that a gigantic XML file quickly becomes unmanageable. This is why
- it is recommended to split monolithic configurations into several files and leverage
-  Mule's capacity to load multiple configuration files at application start-up time.
- Moreover, splitting configurations into multiple fragments encourages re-use across teams.
+Though it seems convenient to keep your Mule configurations in one file,
+a very large XML file quickly becomes unmanageable. This is why
+ Mulesoft recommends that you split monolithic configurations into several files, and leverage Mule's capacity to load multiple configuration files at application start-up time.
+ Splitting configurations into multiple files encourages re-use across teams.
 
-Mule offers two options for loading several configuration files:
+Mule offers two options for loading multiple configuration files:
 
-* side-by-side: provide a list of independent configuration files to load, +
-* imported: have one configuration file import several others, which in-turn can
+* Side-by-side: Provide a list of independent configuration files to load.
+* Imported: Have one configuration file import several others, which in-turn, can
```

Comment:
"in-turn" -> "in turn" 

https://www.merriam-webster.com/dictionary/in%20turn:
in turn
: in due order of succession

Reply:
OK